yarn-project/noir-contracts/src/libs/noir-aztec/src/note/note_getter_options.nr

```diff
@@ -50,7 +50,7 @@ impl<Note, N, S, P> NoteGetterOptions<Note, N, S, P> {
     }
 
     fn with_filter(
-        filter: fn ([Note; MAX_READ_REQUESTS], P) -> [Note; S],
```

Comment:
hmm, this seems like it should still have MAX in the name, not sure how to reconcile that

Reply:
maybe if some of the constants went into a `limits` crate and we had `limits::READ_REQUESTS_LENGTH`